lib/win32/certstore.rb

```diff
@@ -78,6 +78,18 @@ def get(certificate_thumbprint)
       cert_get(certificate_thumbprint)
     end
 
+    # Returns a filepath to a PKCS12 container. The filepath is in a temporary folder so normal housekeeping by the OS should clear it.
+    # However, you should delete it yourself anyway.
+    # @param [certificate_thumbprint<string>] Is the thumbprint of the pfx blob you want to capture
```

Comment:
The correct format here would be `@param certificate_thumbprint [String] Is the thumbprint of the pfx blob you want to capture` see https://rubydoc.info/gems/yard/file/docs/Tags.md#param

Reply:
same applies to the other params

Reply:
I dunno where I picked up that whacky formatting from. Thanks for the pointer.